src/openzeppelin/tests/test_erc20.cairo

```diff
@@ -201,10 +226,9 @@ fn test_transfer_from_doesnt_consume_infinite_allowance() {
 
     let recipient: ContractAddress = contract_address_const::<2>();
     let spender: ContractAddress = contract_address_const::<3>();
-    let amount: u256 = u256_from_felt(100);
+    let amount: u256 = u256_from_felt252(100);
     let max_u256: u256 = u256 {
```

Comment:
should we add this as a constant/function until Hadrien's PR is merged upstream?